platforms/shared/configuration/roles/setup/vault/defaults/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+tmp_directory: "{{ lookup('env', 'TMPDIR') | default('/tmp',true) }}"
+
+default:
+  version: "1.7.0"
+  checksum: ""
```

Comment:
arent we using 1.7.1. also newline

Reply:
done.
upgraded vault client to 1.7.1 to match server running 1.7.1
enabled yamllint rule new-line-at-end-of-file in shared/configuration folder